cetmix_tower_yaml/models/cx_tower_yaml_mixin.py

```diff
@@ -197,11 +198,12 @@ def _post_process_yaml_dict_values(self, values):
         for key, value in filtered_values.items():
             # IMPORTANT: Odoo naming patterns must be followed for related fields.
             # This is why we are checking for the field name ending here.
-            # Further checks for the field type are done in _process_m2o_value()
-            if key.endswith("_id"):
+            # Further checks for the field type are done
+            # in _process_related_field_value()
+            if key.endswith("_id") or key.endswith("_ids"):
                 processed_value = self.with_context(
                     explode_related_record=True
-                )._process_m2o_value(key, value, record_mode=False)
+                )._process_related_field_value(key, value, record_mode=False)
```

Comment:
`_process_relation_field_value`

common name for fields (m2o, x2m) - relation fields

Reply:
@GabbasovDinar `relation` is a field type however in this case function name means that function is processing a field  related to the parent function.
Yet this name looks confusing so let's rename it to avoid possible misunderstanding 👍